package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "mirror-folder",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Oops not sure how this got in there. Its already at `1.0.1` - must have done a bad rebase.